### PR TITLE
tests/sweeper: Remove unnecessary len() check

### PR DESCRIPTION
**Pull request by YakDriver**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: aws/resource_aws_waf_web_acl_test.go [hunk 1]
```diff
@@ -73,21 +73,15 @@ func testSweepWafWebAcls(region string) error {
 	})
 
 	if err != nil {
-		errs = multierror.Append(errs, err)
-		// in case work can be done, don't jump out yet
+		errs = multierror.Append(errs, fmt.Errorf("error describing EC2 WAF Web ACLs: %w", err))
```
Comment (bflad) [inline aws/resource_aws_waf_web_acl_test.go:76]: Just WAF 😎 

```suggestion
		errs = multierror.Append(errs, fmt.Errorf("error describing WAF Web ACLs: %w", err))
```

File: aws/resource_aws_waf_web_acl_test.go [hunk 2]
```diff
@@ -73,21 +73,15 @@ func testSweepWafWebAcls(region string) error {
...
 
-	if len(sweepResources) > 0 {
-		// any errors didn't prevent gathering of some work, so do it
-		sweepErr := testSweepResourceOrchestrator(sweepResources)
-
-		if sweepErr != nil {
-			errs = multierror.Append(errs, sweepErr)
-		}
+	if err = testSweepResourceOrchestrator(sweepResources); err != nil {
+		errs = multierror.Append(errs, fmt.Errorf("error sweeping EC2 WAF Web ACL for %s: %w", region, err))
```
Comment (bflad) [inline aws/resource_aws_waf_web_acl_test.go:80]: Samesies

```suggestion
		errs = multierror.Append(errs, fmt.Errorf("error sweeping WAF Web ACL for %s: %w", region, err))
```

File: aws/resource_aws_waf_web_acl_test.go [hunk 3]
```diff
@@ -73,21 +73,15 @@ func testSweepWafWebAcls(region string) error {
...
 	}
 
 	if testSweepSkipSweepError(errs.ErrorOrNil()) {
-		log.Printf("[WARN] Skipping WAF Web ACL sweep for %s: %s", region, errs)
+		log.Printf("[WARN] Skipping EC2 WAF Web ACL sweep for %s: %s", region, errs)
```
Comment (bflad) [inline aws/resource_aws_waf_web_acl_test.go:84]: Samesies

```suggestion
		log.Printf("[WARN] Skipping WAF Web ACL sweep for %s: %s", region, errs)
```

**Review by bflad — APPROVED** [review]
Minor typos, otherwise looks great 🚀

File: aws/resource_aws_waf_web_acl_test.go (hunk 1 above)
Comment (YakDriver) [reply aws/resource_aws_waf_web_acl_test.go]: 🤦

File: aws/resource_aws_waf_web_acl_test.go (hunk 2 above)
Comment (YakDriver) [reply aws/resource_aws_waf_web_acl_test.go]: 🤦 🤦

File: aws/resource_aws_waf_web_acl_test.go (hunk 3 above)
Comment (YakDriver) [reply aws/resource_aws_waf_web_acl_test.go]: 🤦 🤦 🤦 🤦 🤦 🤦 🤦 🤦 🤦 🤦 🤦 🤦 🤦 🤦 🤦 🤦 🤦 🤦 🤦 🤦 🤦 🤦

**Pull request merged**